stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/websocketconnector/Close.java

```diff
@@ -46,7 +46,9 @@
                 @Argument(name = "reason", type = TypeKind.STRING)
         }
 )
-public class Close extends BlockingNativeCallableUnit {
+public class Close
+        extends BlockingNativeCallableUnit { //This is blocking because of the limitations in the transport where
```

Comment:
Can we inline this with class and add a comment on another line.
Also I think it is better to make this a TODO comment. So we can address this later